WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -181,6 +181,9 @@ class NotificationsViewController: UITableViewController, UIViewControllerRestor
 
     override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
         super.traitCollectionDidChange(previousTraitCollection)
+        DispatchQueue.main.async {
```

Comment:
Yeah, I'll add a comment.

Table Header View layout is _messed up_ in the OS. We need this code to run in the next redraw cycle or it will do nothing :(